commit 58ad7d34c28a5ee58cd065bbc6a240430e7211f1
Author: Stefan Wahren <stefan.wahren@i2se.com>
Date:   Sun Oct 21 19:08:29 2018 +0200

    staging: bcm2835-camera: Add hint about possible faulty GPU mem config
    
    As per default the GPU memory config of the Raspberry Pi isn't sufficient
    for the camera usage. Even worse the bcm2835 camera doesn't provide a
    helpful error message in this case. So let's add a hint to point the user
    to the likely cause.
    
    Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c b/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c
index 51e5b04ff0f5..1e171b434e45 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/mmal-vchiq.c
@@ -1624,8 +1624,11 @@ int vchiq_mmal_component_init(struct vchiq_mmal_instance *instance,
 	component = &instance->component[instance->component_idx];
 
 	ret = create_component(instance, component, name);
-	if (ret < 0)
+	if (ret < 0) {
+		pr_err("%s: failed to create component %d (Not enough GPU mem?)\n",
+		       __func__, ret);
 		goto unlock;
+	}
 
 	/* ports info needs gathering */
 	component->control.type = MMAL_PORT_TYPE_CONTROL;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1339_58ad7d34c28a5ee58cd065bbc6a240430e7211f1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1339_58ad7d34c28a5ee58cd065bbc6a240430e7211f1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
